specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json

```diff
@@ -7623,6 +7688,86 @@
           }
         }
       }
+    },
+    "DataWarehouseQueriesSteps": {
+      "description": "The response to a list datawarehouse queries.steps",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DataWarehouseQueriesStepsProperties",
+          "description": "Resource properties."
+        }
+      }
+    },
+    "DataWarehouseQueriesStepsProperties": {
+      "description": "Properties of a datawarehousequeries.",
+      "type": "object",
+      "properties": {
+        "queryId": {
+          "description": "Requested QueryId. of datawarehousequeries",
+          "type": "string",
+          "readOnly": true
+        },
+        "startTime": {
+          "description": "The start time for queries datawarehousequeries",
```

Comment:
Please fix the description as I am assuming that it will be start time for the query step. If its start time for datawarehouse query, it should not be part of step properties.

Reply:
Added Description as Steps Properties